src/Image/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+interface ImageProps {
+    alt: string;
+    globalSettings: any; // TODO: Get rid of any
+    height: string;
+    src: string;
+    title: string;
+    width: string;
+}
+
+const Image = ({ alt, height, src, title, width }: ImageProps) => {
+    return (
+        <picture>
+            <img
+                alt={alt}
+                height={height}
+                src={src}
+                title={title}
+                width={width}
+            />
+        </picture>
```

Comment:
What's the reason for wrapping this in a `picture` element, vs just returning the `img` directly?

Reply:
Lays the groundwork for responsiveness which I'm hoping we're going to add in at some point. Plus semantic web and all.